lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -764,12 +763,12 @@ def destroy_machine(action_handler, machine_spec, machine_options)
 
     def cloudsearch(api_version="20130101")
       @cloudsearch ||= {}
-      @cloudsearch[api_version] ||= AWS::CloudSearch::Client.const_get("V#{api_version}").new
+      @cloudsearch[api_version] ||= Aws::CloudSearch::Client.const_get("V#{api_version}").new
       @cloudsearch[api_version]
     end
 
     def ec2
-      @ec2 ||= AWS::EC2.new(config: aws_config)
+      @ec2 ||= Aws::EC2.new(config: aws_config)
```

Comment:
These methods (`ec2`, elb`, `elasticache`, etc.) which previously defined the clients have been replaced by the dynamically defined methods on [line 744](https://github.com/MsysTechnologiesllc/chef-provisioning-aws/blob/c5d17253559d293b0087bb49075636c5d0c97a63/lib/chef/provisioning/aws_driver/driver.rb#L774). These old methods can be removed and instead the methods like `ec2_client`, `ec2_resource`, `s3_client`, `s3_resource`, etc. should be used